ChaosMod/Effects/db/Peds/PedsSeriousKamikazes.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+	Effect by Moxi based on Killer Clowns by Last0xygen, explosives peds
+*/
+
+#include <stdafx.h>
+
+static std::list<Ped> kamikazeEnemies;
+static std::list<Object> bombObject;
```

Comment:
As bombs and peds are always 1:1, create a helper struct which contains both to better keep track of them and remove bombs too once the ped gets removed

Reply:
I tried to do that, but my knowledge and skills in C++ are lacking. Everytime I try something the kamikazes die as soon as they spawn.

Any tips?

Reply:
Std::map
https://github.com/gta-chaos-mod/ChaosModV/blob/master/ChaosMod/Effects/db/Peds/PedsNailguns.cpp

Reply:
I tried my best. It seems to be working. Hopefully it's somewhat decent! :D